From 50585db8665fc0aa6cb6414c040e6e80c6c23cd8 Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Sat, 13 Jun 2020 20:38:41 -0400
Subject: ALL_ADAFLAGS filter-out Wno-pedantic-ms-format

The flag Wno-pedantic-ms-format does not work with the gnat compiler;
it raises a warning for each use. "gnat1.exe: warning: command-line 
  option '-Wno-pedantic-ms-format' is valid for C/C++/ObjC/ObjC++ 
  but not for Ada".

---
 gcc/ada/gcc-interface/Make-lang.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/ada/gcc-interface/Make-lang.in b/gcc/ada/gcc-interface/Make-lang.in
index acbe2b877ca..fe3020580ba 100644
--- a/gcc/ada/gcc-interface/Make-lang.in
+++ b/gcc/ada/gcc-interface/Make-lang.in
@@ -65,7 +65,7 @@ ADAFLAGS= $(COMMON_ADAFLAGS)
 endif
 
 ALL_ADAFLAGS = \
-  $(CFLAGS) $(ADA_CFLAGS) $(ADAFLAGS) $(CHECKING_ADAFLAGS) $(WARN_ADAFLAGS)
+  $(filter-out -Wno-pedantic-ms-format, $(CFLAGS) $(ADA_CFLAGS) $(ADAFLAGS) $(CHECKING_ADAFLAGS) $(WARN_ADAFLAGS))
 FORCE_DEBUG_ADAFLAGS = -g
 ADA_CFLAGS =
 ADA_INCLUDES = -nostdinc -I- -I. -Iada/generated -Iada -I$(srcdir)/ada -I$(srcdir)/ada/gcc-interface -Iada/libgnat -I$(srcdir)/ada/libgnat
-- 
